testing/python_lib/test_faucetizer.py

```diff
@@ -26,25 +26,31 @@ def __init__(self, *args, **kwargs):
         self._temp_dir = None
         self._temp_structural_config_file = None
         self._temp_behavioral_config_file = None
+        self._temp_segments_vlans_file = None
 
     def _setup_config_files(self):
         self._temp_dir = tempfile.mkdtemp()
         _, self._temp_structural_config_file = tempfile.mkstemp(dir=self._temp_dir)
         _, self._temp_behavioral_config_file = tempfile.mkstemp(dir=self._temp_dir)
+        _, self._temp_segments_vlans_file = tempfile.mkstemp(dir=self._temp_dir)
 
         with open(self._temp_structural_config_file, 'w') as structural_config_file:
             structural_config_file.write(self.FAUCET_STRUCTURAL_CONFIG)
 
+        with open(self._temp_segments_vlans_file, 'w') as segments_vlans_file:
+            segments_vlans_file.write(self.SEGMENTS_TO_VLANS)
+
     def _cleanup_config_files(self):
         shutil.rmtree(self._temp_dir)
 
     def _initialize_faucetizer(self):
         self._orch_config = str_proto(self.ORCH_CONFIG, OrchestrationConfig)
 
         self._faucetizer = Faucetizer(
-            self._orch_config, self._temp_structural_config_file, self.SEGMENTS_TO_VLANS,
+            self._orch_config, self._temp_structural_config_file,
             self._temp_behavioral_config_file)
         self._faucetizer.reload_structural_config()
+        self._faucetizer.reload_segments_to_vlans(self._temp_segments_vlans_file)
```

Comment:
Does anything in here actually check that the file was automatically reloaded when it changes on disk, or is it only testing that it can load the file from disk?  I would expect some sequence like: setup initial file, check results, modify file, check results have changed.

Reply:
This is more like a unit test that purely test faucetizer itself. The file reload is managed by the file watcher. I think I can add it to the DVA integration test.